### PR TITLE
[CMakeV1] Made the task to be executed through shell.

### PR DESCRIPTION
**Pull request by egor-bryzgalov**
Related to issue #12848

Made the task to be executed through shell specific for the environment to handle variables in command string properly.

**NOTE:** [These changes in azure-pipelines-task-lib](https://github.com/microsoft/azure-pipelines-task-lib/pull/637) are required.

### CODE REVIEW

**Comment by jtpetty** [comment]
@egor-bryzgalov - There seems to be some issues with CI for this PR. Once all the tests are passing, I will review the PR.

**Comment by jtpetty** [comment]
Also, @egor-bryzgalov, if you need assistance figuring out what is going on, please feel free to reach out and I can help dig in with you.

**Comment by damccorm** [comment]
You're going to have to update to the most recent version of the task-lib. Since its currently pinned to `^2.9.3` in the package.json and `2.9.3` in the package-lock, it won't get updated (since 2.9.3 satisfies the ^2.9.3 requirement). To update you can just change it to `^2.9.5` in the package.json and run `npm install`

File: Tasks/CMakeV1/package.json [hunk 1]
```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "azure-pipelines-task-lib": "^2.9.3"
+    "azure-pipelines-task-lib": "^2.9.5"
```
Comment (damccorm) [inline Tasks/CMakeV1/package.json:20]: Nit: Run npm install in this directory to get the changes to show up in the package-lock as well. Alternately, run `node make.js build --task CMakeV1` to build this task which will have the same effect

**Comment by damccorm** [comment]
@jtpetty this should be ready for review from your team

File: Tasks/CMakeV1/package.json (hunk 1 above)
Comment (egor-bryzgalov) [reply Tasks/CMakeV1/package.json:20]: Thank you! Got it done.

**Pull request merged**